### PR TITLE
Adds a range of reserved usernames to reduce potential misuse by malicious actors

### PR DESCRIPTION
**Pull request by jmking-iftas**
Adds a range of reserved usernames to reduce potential misuse by malicious actors

alpha sort for readability

new reservations:
    - abuse
    - account
    - accounts
    - administration
    - admins
    - helpdesk
    - instance
    - moderators
    - mods
    - owner
    - security
    - server
    - staff
    - support

### CODE REVIEW

**Comment by shleeable** [comment]
Excellent default list.

future enhancements might be offering an webmin managed list (with a role to allow mods to modify)... or even modified via API.

**Comment by renchap** [comment]
Related: https://github.com/mastodon/mastodon/issues/32023 which has been tagged as open for contributors

**Review by Gargron — DISMISSED** [review]

**Comment by shleeable** [comment]
why is support twice?

**Comment by ClearlyClaire** [comment]
This needs changes to the tests, as some were using account names that are now disallowed.

**Comment by mjankowski** [comment]
When https://github.com/mastodon/mastodon/pull/32856 merges and this one is rebased, I think the specs here will pass.

Will watch CI to see if I missed something.

File: config/settings.yml [hunk 1]
```diff
@@ -21,14 +21,28 @@ defaults: &defaults
...
     - moderator
+    - moderators
+    - mods
+    - owner
+    - root
+    - security
+    - server
+    - staff
+    - support
+    - support
```
Comment (mjankowski) [inline config/settings.yml:44]: Dupe value of "support"?

Thank you for alphabetizing this list.

**Review by ClearlyClaire — APPROVED** [review]

**Pull request merged**